release-blocking-jobs.md

```diff
@@ -2,56 +2,122 @@
 
 Here is a list of jobs that should block release if not healthy.
 
-If you want to add a release-branch job, please add them to sig-release-x.x-all dashboard first,
-and show the test is not flaky, then the release team can decide if the test should be added
-to release-blocking dashboard as well.
+If you want to add a release-branch job, please add them to the appropriate
+sig-release-x.y-all dashboard first. Then show the test is not flaky, and the
+release team can decide if the test should be added to release-blocking 
+dashboard as well.
 
-<!-- TODO(krzyzacy): define blocking criteria -->
+## Release Blocking Criteria
+
+The ideal release-blocking job meets the following criteria:
```

Comment:
what if an already-blocking job doesn't, do we kick it out of release blocking?

Reply:
My goal is for us to use followup PR's to suggest kicking out of blocking.  If we decide no, we can't kick out just yet, we'll at least have a documented decision for why it needs to stay

Reply:
sgtm, maybe also document that here (when/who will need to make that followup kickout PR)

Reply:
opened https://github.com/kubernetes/sig-release/issues/347